projects/canopy-test-app/src/app/app.component.html

```diff
@@ -567,6 +567,17 @@
               </tbody>
             </table>
 
+            <lg-heading level="3">Modal</lg-heading>
+
+            <button lgModalTrigger="modal-story" lg-button type="button" variant="outline-primary">Open modal</button>
+            <lg-modal id="modal">
```

Comment:
I think the `id` here needs to match the `lgModalTrigger` value above?

Reply:
doh yes ;)

Reply:
Updated 👍